metabase/metabase
# Add new sample Analytic Events and Feedback tables

Epic https://github.com/metabase/metabase/issues/24807

How to test:
- Open Admin -> Data Model
- Make sure there are new `Analytic Events` and `Feedback` tables
- Make them visible
- Go to the home page and make sure x-rays for these table work

If a table isn't visible, you might need to trigger manual schema sync in Admin -> Databases.

<img width="1728" alt="Screenshot 2022-08-22 at 17 20 51" src="https://user-images.githubusercontent.com/8542534/185944454-53f6a751-840e-469c-975e-a17ac0c0a914.png">
